src/components/CodeField.tsx

```diff
@@ -121,37 +150,22 @@ export const JsonField: FC<FieldProps<JsonFieldSchema>> = ({
                 )}
               </Flex>
               <Collapse in={isOpen} style={{ overflow: 'visible' }}>
-                <JSONInput
-                  id={`json-input__${name}`}
-                  theme="light_mitsuketa_tribute"
-                  locale={locale}
-                  reset={false}
-                  placeholder={getPlaceholder()}
+                <Editor
+                  onMount={handleEditorDidMount}
+                  defaultValue={getPlaceholder()}
+                  theme={colorMode.colorMode === 'light' ? 'light' : 'vs-dark'}
                   height={height ?? '200px'}
-                  width="100%"
-                  viewOnly={isReadOnly || disabled || readOnly}
-                  onChange={(value: { jsObject: any }) => {
-                    setValue(
-                      name,
-                      stringify
-                        ? JSON.stringify(value.jsObject)
-                        : value.jsObject
-                    );
+                  defaultLanguage={language}
+                  options={{
+                    minimap: { enabled: false },
+                    readOnly: isReadOnly || disabled || readOnly,
+                    autoClosingBrackets: 'always',
+                    autoClosingOvertype: 'always',
+                    automaticLayout: true,
+                    tabCompletion: 'on',
```

Comment:
I think we should bump the font-size here: `fontSize: 16` will match the other inputs

Reply:
👍 
sure thing